index.js

```diff
@@ -30,8 +30,11 @@ export { default as withKiwtFieldArray } from './lib/withKiwtFieldArray';
 
 // Functions, utilities, and misc.
 export { default as customPropertyTypes } from './lib/customPropertyTypes';
+export { default as EResourceType } from './lib/EResourceType';
```

Comment:
Components go in the first block per file style.

Reply:
> It'd be a good idea to write tests for these utilities. These things that pull in data from various sources are tough to test outside of unit tests. Additionally, we're already planning on refactoring this stuff in the future so tests allow us to confidently refactor without being afraid that we're missing something.

Sounds good

Reply:
Updated the PR with utility helpers tests.  Just FYI: `getResourceIdentifier` and ``getSiblingIdentifier` are currently being called in only one [context](https://github.com/folio-org/ui-agreements/blob/master/src/components/views/EResources.js#L231), whereas `EresourceType` is being called from 6 different contexts. All of them should be covered by these tests